test/processinfo.js

```diff
@@ -32,6 +32,8 @@ t.test('build some processinfo', t => {
   proc.stderr.resume()
   proc.stdout.resume()
   proc.on('close', (code, signal) => {
+    // The subprocess is failing with the following error:
+    // " External ID blorp used by multiple processes"
```

Comment:
I was able to determine this by changing the preceding lines to the following:

```diff
-  proc.stderr.resume()
-  proc.stdout.resume()
+  proc.stderr.pipe(process.stderr)
+  proc.stdout.pipe(process.stdout)
```

This let me see the actual error message from the subprocess. This error actually comes from `istanbul-lib-processinfo`: https://github.com/istanbuljs/istanbul-lib-processinfo/blob/b9cd52cf6eb140cf3bff24ba5df34d5812845d66/index.js#L250

I have no idea what `blorp` is, what `NYC_PROCESSINFO_EXTERNAL_ID` is supposed to do, or why it would end up being used by multiple processes (though this probably has something to do with the watchdog process that `foreground-child` creates).